### ISSUE TITLE
All new boxes hang on vagrant up "Selecting on IO"

### ISSUE THREAD

**Issue by Saoirse1916** [puphpet/puphpet#2503]
After my boxes stopped working with this problem (https://github.com/puphpet/puphpet/issues/2501) the comment suggested that the problem was resolved with the latest versions, so I rebuilt and downloaded a new box.  However, now it simply hangs on vagrant up and when I run --debug the last lines are:

 INFO vmware_driver: Enabling shared folders...
 INFO subprocess: Starting process: ["/Applications/VMware Fusion.app/Contents/Library/vmrun", "enableSharedFolders", "/Users/Sean/Downloads/relynx/.vagrant/machines/relynx/vmware_fusion/a78ac259-1857-4e07-8b04-cd29d7e75953/centos-6.8-x86_64.vmx"]
DEBUG subprocess: Command not in installer, not touching env vars.
 INFO subprocess: Command not in installer, restoring original environment...
DEBUG subprocess: Selecting on IO

Here's my config.yaml:
http://pastebin.com/wRScB2B3

EDIT: I tried this with both NFS and Default file systems, same error.

EDIT 2: I then tried again stripping out all shared folders and all databases, same error.  Here's the config.yaml for that one: http://pastebin.com/Zb6cAeSa

**Comment by jtreminio** [comment]
The VMWare boxes were built against VMWare Fusion 8.5 Pro.

I _believe_ they work, but I do not have a working vagrant-vmware plugin to actually test them out with Vagrant. I know they spin up outside of Vagrant thouhg.

**Comment by Saoirse1916** [comment]
I just updated to VMWare Fusion 8.5 and then updated the vagrant-vmware plugin to match.  Now the machine spins up but I'm back to the same error I got in the previous issue: 

Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.

So the machine is up and is pingable, but I can't ssh and nothing works.
